repo/packages/G/grafana/7/package.json

```diff
@@ -0,0 +1,23 @@
+{
+    "packagingVersion": "4.0",
+    "upgradesFrom": [
+        "*"
+    ],
+    "downgradesTo": [
+        "*"
```

Comment:
Shouldn't this be the previous package version?

Reply:
I just copied this from 6/package.json. Should I change it to 3.0?